File: src/UE4_Packages/UInterfaceProperty.java
/**
 * 
 */
package UE4_Packages;

/**
 * @author FunGames
 *
 */
public class UInterfaceProperty {
	private long interfaceNumber;

	public long getInterfaceNumber() {
		return interfaceNumber;
	}

	
	public UInterfaceProperty(FArchive Ar) throws ReadException {
		interfaceNumber = Ar.readUInt32();
	}
	
}

File: src/main/kotlin/me/fungames/jfortniteparse/ue4/objects/uobject/UInterfaceProperty.kt
package me.fungames.jfortniteparse.ue4.objects.uobject

import me.fungames.jfortniteparse.ue4.reader.FArchive
import me.fungames.jfortniteparse.ue4.writer.FArchiveWriter

class UInterfaceProperty {
    var interfaceNumber: UInt

    constructor(Ar: FArchive) {
        interfaceNumber = Ar.readUInt32()
    }

    fun serialize(Ar: FArchiveWriter) {
        Ar.writeUInt32(interfaceNumber)
    }

    constructor(interfaceNumber: UInt) {
        this.interfaceNumber = interfaceNumber
    }
}